Makefile.common

```diff
@@ -6,16 +6,22 @@ SOURCES = $(SOURCES_C) $(SOURCES_ASM)
 OBJECTS = $(SOURCES_C:.c=.o) $(SOURCES_ASM:.S=.o)
 DEPFILES = $(SOURCES_C:%.c=.%.D) $(SOURCES_ASM:%.S=.%.D)
 
-TRIPLET_CUSTOM  = mipsel-unknown-elf-
-TRIPLET_IMGTECH = mips-mti-elf-
-
-CUSTOM_TOOLCHAIN_FOUND = $(shell which $(TRIPLET_CUSTOM)gcc > /dev/null; echo $$?)
-# If custom toolchain is available, use it. Otherwise fall back to ImgTech's.
-ifeq ($(CUSTOM_TOOLCHAIN_FOUND),0)
-	P = $(TRIPLET_CUSTOM)
-else
-	P = $(TRIPLET_IMGTECH)
+TRIPLET = mipsel-mimiker-elf
+TRIPLET_OLD = mipsel-unknown-elf
+
+TOOLCHAIN_FOUND = $(shell which $(TRIPLET)-gcc > /dev/null; echo $$?)
+OLD_TOOLCHAIN_FOUND = $(shell which $(TRIPLET_OLD)-gcc > /dev/null; echo $$?)
+ifneq ($(TOOLCHAIN_FOUND), 0)
+  ifneq ($(OLD_TOOLCHAIN_FOUND), 0)
+    $(error "mips-mimiker-elf" toolchain not found)
+  else
+    # Old toolchain is present, but the new one is not. Instruct the developer.
+    $(error You are using "mips-unknown-elf" toolchain. Please switch to "mips-mimiker-elf")
```

Comment:
It's probably better to ask developer to re-read `README.md` in previous branch, rather than explicitely checking for old toolchain.